tests/FakeItEasy.Specs/ArgumentCapturingSpecs.cs

```diff
@@ -257,13 +257,13 @@ public static void CaptureNegatedConstrainedValues(Action<int> fake, Captured<in
 
         [Scenario]
         public static void CaptureMutatingArgument(
-            Action<IList<int>> fake, IList<int> listOfInts, Captured<IList<int>> capturedArgument)
+            Action<IList<int>> fake, IList<int> listOfInts, Captured<IList<int>, List<int>> capturedArgument)

```

Comment:
Doesn't this cause [CA1002](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1002)?

Reply:
Oh. The `List<T>` isn't directly a parameter, it's a type argument of a parameter, which is probably fine...

Reply:
I see no evidence of the warning in VS nor the CI builds&hellip;

Reply:
Yeah, I checked the logs and didn't see it. That's what prompted me to look closer...